octobot_trading/api/symbol_data.py

```diff
@@ -39,6 +40,72 @@ def get_symbol_klines(symbol_data, time_frame) -> list:
     return symbol_data.symbol_klines[TimeFrames(time_frame)].kline
 
 
+def get_symbol_close_candles(symbol_data, time_frame, limit=-1, include_in_construction=False):
```

Comment:
I don't think these methods implementations should be there. Probably optimized in another file and compiled in cython to keep using low level numpy api.

Reply:
What do you think about it ?

Reply:
Right, I'm moving them